<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="atlas" >
  <xacro:include filename="../components/atlas_v5_convex_hull.urdf" />
  <xacro:include filename="../../common_components/multisense_sl/multisense_sl_convex_hull.urdf" />
  <xacro:include filename="./atlas_v5_common.xacro" />
  <xacro:include filename="../../common_components/robotiq_hand_description/robotiq_hand_convex_hull.xacro" />

  <!-- Set the initial hand position and orientation by changing 'rpy' and 'xyz'
  to mach the values in your actual robot. -->
  <xacro:robotiq_convex_hull side="left"  is_open="0" use_visuals="1" parent="l_hand_force_torque"/>
  <xacro:robotiq_convex_hull side="right" is_open="0" use_visuals="1" parent="l_hand_force_torque"/>

  <xacro:robotiq_camera  side_letter="l" reflect="1" parent="l_hand_force_torque" >
    <origin xyz="0.0  0.0 0.0" rpy="0.0 0  0.0"/>
  </xacro:robotiq_camera>
  <xacro:robotiq_camera  side_letter="r" reflect="-1" parent="r_hand_force_torque" >
    <origin xyz="0.0  0.0 0.0" rpy="0.0 0  0.0"/>
  </xacro:robotiq_camera>
</robot>
